src/event_loop.rs

```diff
@@ -359,6 +361,16 @@ impl<T> EventLoopWindowTarget<T> {
         #[cfg(any(x11_platform, wayland_platform, windows))]
         self.p.set_device_event_filter(_filter);
     }
+
+    /// Returns the current cursor position in screen coordinates.
+    ///
```

Comment:
Could you document what the difference between `Window::cursor_position` and `EventLoopWindowTarget::cursor_position` is? Or are they exactly the same? And if they're the same, perhaps we can get by with just one of them?

Reply:
yeah, they are the same, and is provided on `Window` for convenience purposes only.

Reply:
the new function stands out like a sore thumb here...
and it opens a door to many more convenience functions.
do we want to go that down that road.

Reply:
Well, there is `primary_monitor` and `available_monitors` which are provided on `Window` for convenience too. I just followed the pattern, but I don't have a strong feeling for it and removing it, is fine with me.

Reply:
removed it from `Window` as I couldn't think of an immediate use-case for it.